src/data/polynomial/degree/definitions.lean

```diff
@@ -1047,10 +1047,11 @@ calc degree (p - q) = degree (erase (nat_degree q) p + -erase (nat_degree q) q)
   : degree_neg (erase (nat_degree q) q) ▸ degree_add_le _ _
 ... < degree p : max_lt_iff.2 ⟨hd' ▸ degree_erase_lt hp0, hd.symm ▸ degree_erase_lt hq0⟩
 
+lemma degree_X_sub_C_le {r : R} : (X - C r).degree ≤ 1 :=
```

Comment:
Maybe it is better with `r` explicit?

Reply:
I make it implicit because it's implicit in the existing lemma `nat_degree_X_sub_C_le`.

Reply:
I think it should be explicit in both lemmas. Do you mind making this modification? `nat_degree_X_sub_C_le` is not used a lot, so it should be very easy.

Reply:
Done.